docs/products/kafka/howto/configure-with-kafka-cli.rst

```diff
@@ -0,0 +1,53 @@
+Manage configurations with Apache Kafka® CLI tools
+==================================================
+
+ZooKeeper is an important part of Apache Kafka® since it tracks the status of the cluster and offers visibility its the current state. Moreover, by interacting with ZooKeeper, you're able to change any Apache Kafka parameter.
+
+Aiven for Apache Kafka® is a fully managed service, therefore, to guarantee the service stability, good maintenance and functionality, we do not allow direct access to ZooKeeper. Instead, we monitor and manage it using the Aiven platform to make sure that we can react as quickly as possible if any issues occur.
+
+Most of the changes usually performed via ZooKeeper are available via the :doc:`advanced configuration flags in the Aiven web console <use-zookeeper>` or via the :doc:`Aiven CLI </docs/tools/cli>`. However, you may want to use the client tools included in the `Apache Kafka binaries <https://kafka.apache.org/downloads>`_ to manage the configurations for your services. 
+
+.. Warning::
+    
+    Without access to ZooKeeper, some of the Apache Kafka command line tools are not working. But some commands, with the Apache Kafka product maturation, removed the reliance on ZooKeeper giving you options to handle specific configuration changes.
```

Comment:
This is not very helpful. Do we know what does or doesn't work? I think it might be better to remove it and cover this in the recommendation to use Aiven's tools where appropriate.